docs/home/howto/CLUSTER.md

```diff
@@ -464,7 +466,16 @@ To setup the cluster do the following steps from the provisioning machine:
 
     Epicli will read this file and automaticly use it for authentication for resource creation and management.
 
-    For both `aws`and `azure` there is a `use_public_ips` tag. When this is true the VM's will also have a direct inferface to the internet. While this is easy for setting up a cluster for testing it should not be used in production. A VPN setup should be used which we will document in a different section (TODO).
+    For both `aws`and `azure` the following cloud attributes overlap:
+    - `use_public_ips`: When `true`, the VMs will also have a direct interface to the internet. While this is easy for setting up a cluster for testing it should not be used in production. A VPN setup should be used which we will document in a different section (TODO).
+    - `os_image`: Lets you more easily select Epiphany team validated and tested OS images. When one is selected, it will be applied to **every** `infrastructure/virtual-machine` document in the cluster regardless of user defined ones.
+                  The following values are accepted:
+                  - `default`: Applies user defined `infrastructure/virtual-machine` documents when generating a new configuration.
```

Comment:
IMO the `default` world can be misleading in this context since `user defined` looks more like `custom` than `default`. 
So the first idea is:
1) Rename `default` to `custom`.
2) Use `ubuntu-18.04-x86_64` as the default value for this setting.

Second idea could be:
1) Rename `os_image` property to `default_os_image` or `global_os_image` or `force_os_image`.
2) If property from point 1 is present, apply its value, otherwise use user defined `infrastructure/virtual-machine` documents.

This is just my personal thoughts not a request so feel free to ignore it 😉

Reply:
I think first idea is not really clean but I think renaming os_image to default_os_image would be a good solution.

Reply:
Done.